oak-search/src/main/java/org/apache/jackrabbit/oak/plugins/index/search/spi/query/FulltextIndex.java

```diff
@@ -92,6 +92,12 @@
      */
     protected abstract boolean filterReplacedIndexes();
 
+    /*
+    * Whether the isActiveIndex check should run during filtering of replaced indexes.
+    *
+     */
+    protected abstract boolean checkIsIndexActive();
```

Comment:
Not sure if the name of the method is clear enough... Alternatives:

* checkWhetherIndexIsActive()
* areAllIndexesActive()

Just an idea...

Reply:
I had the same thought. Not sure what could be a more meaningful name.

Reply:
reading this again... checkWhetherIndexIsActive() is just longer but not clearer. areAllIndexesActive() seems better. what about areAllIndexesOfThisTypeActive() ? is it very long, but I think that's fine... and it sounds easier to understand

Reply:
areAllIndexesOfThisTypeActive() -> This might not suggest what the method tries to do here - this suggests that this method tells whether the indexes of this type are active or not, whereas in reality
this method basically tells whether to run he isIndexActive check (https://github.com/oak-indexing/jackrabbit-oak/blob/trunk/oak-search/src/main/java/org/apache/jackrabbit/oak/plugins/index/search/spi/query/IndexName.java#L211) or not. 


If checkIsIndexActive() is not clear ,  I suggest 

runIsActiveIndexCheck() or shouldRunIsActiveIndexCheck() instead.

@thomasmueller @fabriziofortino  - wdyt ?

Reply:
If i can pitch in, `runIsActiveIndexCheck()` seems good to me :)